src/main/resources/messages_es.properties

```diff
@@ -963,7 +969,7 @@ ebtcard-subheader=Si lo aprueban para recibir beneficios del Programa de Asisten
 
 authorized-rep.title=Representante autorizado
 authorized-rep.header=�Le gustar�a asignar a una persona como su representante autorizado?
-authorized-rep.subheader=Esta persona puede ayudarte con tus beneficios o actuar en tu nombre. Esto significa que podr�an:
+authorized-rep.subheader=Esta persona puede ayudarle con sus beneficios o actuar en su nombre. Esto significa que podr�an:
```

Comment:
Are these added in transifex as well? it's easy to see what still needs translations and where translations can be reused